RecoTracker/SpecialSeedGenerators/interface/CtfSpecialSeedGenerator.h

```diff
@@ -67,6 +67,14 @@ class CtfSpecialSeedGenerator : public edm::stream::EDProducer<> {
 
 private:
   edm::ParameterSet conf_;
+
+  const edm::ESGetToken<MagneticField, IdealMagneticFieldRecord> mfToken;
```

Comment:
```suggestion
  const edm::ESGetToken<MagneticField, IdealMagneticFieldRecord> theMFToken;
 ```
`the` seems to be the more common prefix for the data members in this class; a trailing underscore is another option.

Reply:
done.